src/ray/core_worker/core_worker.h

```diff
@@ -686,6 +698,9 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
   /// Whether local reference counting is enabled.
   const bool ref_counting_enabled_;
 
+  /// Is local mode being used.
+  const bool local_mode_enabled_;
```

Comment:
`is_local_mode_`  sounds better here, since it's not a functionality to be enabled, but instead an execution mode to switch to.